src/main/resources/db/migration/V0.0.2_2015.10.24_01_01__tasks_list_table_created.sql

```diff
@@ -0,0 +1,6 @@
+CREATE TABLE tasks_list (
+	id BIGINT auto_increment,
+	NAME VARCHAR(100),
+	board_id BIGINT,
+	KEY (id)
```

Comment:
I might be wrong, but I believe that KEY is not the same as PRIMARY KEY in MySQL